src/annotations/components/annotationForm/AnnotationForm.tsx

```diff
@@ -29,24 +29,45 @@ type AnnotationType = 'point' | 'range'
 
 interface Props {
   startTime: string
+  endTime?: string
   title: 'Edit' | 'Add'
   type: AnnotationType
   onSubmit: (Annotation) => void
   onClose: () => void
 }
 
+export const checkAnnotationFormValidity = (
+  annotationType: string,
+  message: string,
+  startTime: any,
+  endTime: any
+) => {
```

Comment:
what does this function return? looks like a boolean? can you type hint that please?